ext/fuzztest.cmd

```diff
@@ -9,9 +9,9 @@
 
 git clone https://github.com/google/fuzztest.git
 cd fuzztest
-: # There is no tagged release as of 2024/01/26. Pick the earliest commit that fixes the
-: # undefined reference to LLVMFuzzerRunDriver when building ubsan tests.
-git checkout a53a2083e7df08749ea26b5960c05a9bffa186c2
+: # There is no tagged release as of 2024/01/26. Pick the earliest commit that fixes
+: # a validation bug in FlatMapImpl.
```

Comment:
I think we can delete this comment, or replace it with a general statement that we use a stable snapshot.